### PR TITLE
feat: add docs issue template

### PR DESCRIPTION
**Pull request by wsocha**
# Description

Please include a summary of the change and which issue is fixed. 

If you are a Boxer, please also reference the related `DDOC` or `APIWG` tickets.
If you do not have a related `Jira` ticket and this is more than a bug fix, then
create one now.

Fixes # (issue)
Re `DDOC-#` (`Jira` ticket)
Re `APIWG-#` (`Jira` ticket)

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own changes
- [ ] I have run `yarn lint` to make sure my changes pass all linters
- [ ] I have pulled the latest changes from the upstream developer branch

## Contribution guidelines

For contribution guidelines, styleguide, and other helpful information please
see the `CONTRIBUTING.md` file in the root of this project.

### CODE REVIEW

**Review by antusus — APPROVED** [review]
👍

File: .github/ISSUE_TEMPLATE/docs_error.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: API documentation error
+description: Report a typo or error found in API documentation
+labels: ["documentation"]
+body:
+  - type: textarea
+    attributes:
+      label: Description of the issue
+      description: Description of what issue you have discovered. Please include as to why you think the issue is an issue.
```
Comment (superojla) [inline .github/ISSUE_TEMPLATE/docs_error.yaml:8]: I'd rephrase _Description of what issue you have discovered. Please include as to why you think the issue is an issue._ to _Describe what you discovered and explain why it is an issue._

File: .github/ISSUE_TEMPLATE/docs_error.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  - type: textarea
+    attributes:
+      label: Current content
+      description: What does the specification currently say?
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Expected content
+      description: What do you think the specification should say instead?
```
Comment (superojla) [inline .github/ISSUE_TEMPLATE/docs_error.yaml:20]: I'd rephrase _What do you think the specification should say instead?_ to _What should the specification say instead?_

**Review by superojla — COMMENTED** [review]
Two comments regarding wording
